content/techniques/queues.md

```diff
@@ -408,6 +408,40 @@ BullModule.forRootAsync({
 
 This construction works the same as `useClass` with one critical difference - `BullModule` will lookup imported modules to reuse an existing `ConfigService` instead of instantiating a new one.
 
+#### Separate processes
+
+This module allows you to run your job handlers in fork processes.
+To do so, add the filesystem path to a file (or more) exporting your processor function to the `processors` property of the BullModule options.
+You can read more on this subject in Bull's [documentation](https://github.com/OptimalBits/bull#separate-processes).
+
+Please note that, your function being executed in a fork, Nestjs' DI won't be available.
+
+```ts
+// app.module.ts
+import { Module } from '@nestjs/common';
+import { BullModule } from 'nest-bull';
+import { join } from 'path';
+
+@Module({
+  imports: [
+    BullModule.forRoot({
+      processors: [join(__dirname, 'processor.js')],
+    }),
+  ],
+})
+export class AppModule {}
+```
+
+```ts
+// processor.ts
+import { Job, DoneCallback } from 'bull';
+
+export default function (job: Job, cb: DoneCallback) {
+  console.log(`[${process.pid}] ${JSON.stringify(job.data)}`);
+  cb(null, 'It works');
+}
```

Comment:
Maybe this is me not knowing Bull too well, but how does Bull decide which function to run, if there are several under `processors`? Does it have some sort of way to map it, or does that need to be handled in this method?

Reply:
No problem @jmcdo29, Bull doesn't support broadcast as we can see in this issue: https://github.com/OptimalBits/bull/issues/1392.

So is not possible to have multiple processors for the same queue, you can although declare multiples processors for named jobs, which is (_I believe_) the case for processors's type be an array of `BullQueueProcessor` and not a single one.

Hope to have answered your questions